### PR TITLE
Use dignified.js

### PR DESCRIPTION
**Pull request by dignifiedquire**
https://github.com/dignifiedquire/dignified.js/issues/1

### CODE REVIEW

**Comment by xicombd** [comment]
LGTM :+1:

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const gulp = require('gulp')
+const multiaddr = require('multiaddr')
+const WSlibp2p = require('./src')
+
+let ws
+
+gulp.task('test:browser:before', (done) => {
```
Comment (diasdavid) [inline gulpfile.js]: Are these `before` and `after` scripts something that `aegir` expects?

**Comment by diasdavid** [comment]
I'm not sure if I understand how the magic works. Could you write how do you organize browser tests that require a Node.js part as well?

@jgantunes this will be interesting for you for your spdy PR https://github.com/diasdavid/js-libp2p-spdy/pull/7

**Comment by diasdavid** [comment]
Merged cause the lack of the `jsnext:main` pointer was causing a bug on libp2p-swarm

**Comment by diasdavid** [comment]
## Important

@dignifiedquire when running `npm run release`, the browser tests fail (because dignified-release doesn't use the patched version of the gulp script). I actually managed to release it by accident because when I hit ctrl+c to stop it, it canceled that test and jumped to the next point of the script, which is 'release', I believe that this behaviour is also not expected.

**Pull request merged**